library/include/hipblas.h

```diff
@@ -19276,16 +19676,20 @@ HIPBLAS_EXPORT hipblasStatus_t hipblasAxpyStridedBatchedEx(hipblasHandle_t   han
     @param[in]
     x         device pointer storing vector x.
     @param[in]
-    xType [hipblasDatatype_t]
-           specifies the datatype of vector x.
+    xType     [hipblasDatatype_t] [DEPRECATED]
+              specifies the datatype of vector x.\n
+    xType     [hipDataType]
+              specifies the datatype of vector x.
     @param[in]
     incx      [int]
               specifies the increment for the elements of y.
     @param[in]
     y         device pointer storing vector y.
     @param[in]
-    yType [hipblasDatatype_t]
-          specifies the datatype of vector y.
+    yType     [hipblasDatatype_t] [DEPRECATED]
+              specifies the datatype of vector y.\n
+    yType     [hipDataType]
```

Comment:
Newlines \n showing up.

Reply:
For doxygen formatting, should create a newline in the docs. I changed it slightly in the next commit to look better. Note that if you're looking at CI docs, the API is erroneously under "guidelines", I'll make a ticket for it.

Reply:
Okay I found it under guidelines and it looks okay.   Maybe in that ticket you can push solver API after blas extensions.  Also will the hipblasDataType enum be deprecated in the docs in that PR or this one?

Reply:
Sure, will do.
Should have been in this or the last PR, but I have another one coming with changes to gemmEx so I'll just put it in that one.